spec/models/bookings/data/school_updater_spec.rb

```diff
@@ -31,6 +31,7 @@
         address_2: "North Kensington",
         town: "London",
         postcode: "W10 5EF",
+        website: "http://www.st.rbkc.sch.uk",
```

Comment:
Do we need to default to `http` @martyn-w?

This website and the others that are being updated in this PR are all https sites in reality, yet we are pointing to http (and the user is then redirected)

We should definitely point to https if it exists - do any schools only have an http site?

![image](https://github.com/user-attachments/assets/57b2007b-407c-4d37-a040-a68a06fdd83b)